ChangeLog.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 10/14/2019 0.19.8
+
+- Add an optional logging callback to consume by livevalidator in order to dump log to external caller.
+
+## 10/12/2019 0.19.7

```

Comment:
I didn't bump the version to 0.19.7 for those 2 change. 
Could you merge them together with 0.19.8

Reply:
Updated.